Sources/Sentry/SentrySDK.m

```diff
@@ -293,6 +293,31 @@ + (void)installIntegrations
     }
 }
 
+/**
+ * Closes the SDK and uninstalls all the integrations.
+ */
++ (void)close
+{
+    // pop the hub and unset
+    SentryHub *hub = SentrySDK.currentHub;
+    [SentrySDK setCurrentHub:nil];
```

Comment:
I wonder what consequences this could have like code that null checked for the hub bound to the scope and goes ahead using it while accessing it through this getter.
Probably best to bind a NoOp Hub here instead though this repo doesn't have one yet. Maybe we just need to validate it's safe to set this to null mid-flight.

Reply:
if I understand correctly, `currentHub` is actually a method/property, but it shadows a static global with the same name.
Anyhow, the method does a synchronized null check and creates an empty Hub in that case.